cinnabar/measurements.py

```diff
@@ -52,19 +54,52 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        IC50: float,
                         label: str,
+                        uncertainty: float = 0,
                         source: str = '',
+                        temperature: float = 298,
                         ):
         """Create Measurement from experimental data
 
         Parameters
         ----------
+        IC50: float
+          experimental IC50 value in unit nM
+        label: str, optional
+          label for this data point.
+        uncertainty: float, optional
+          uncertainty of the experimental value in nM, default is zero if no uncertainty is provided
+        source: str, optional
+          source of experimental measurement
+        temperature: float
+          temperature in K at which the experimental measurement was carried out. By default: 298 K
         """
+        R = 1.9872042586408 * 0.001 * unit.kilocalorie_per_mole / unit.kelvin # Gas constant in kcal/mol/K
+        IC50_upper = IC50 * unit.nanomolar + uncertainty * unit.nanomolar
```

Comment:
This is a different approach to how we do it in the PLB, I would suggest using that approach so we're consistent unless this one has some kind of advantage.

Reply:
@richardjgowers Looking at the code in the PLB, there are conversion between pKi, Ki, IC50 and DG. Would it make sense to use some of that code/extend the conversion here or do you think this is enough (once fixed)?
https://github.com/openforcefield/protein-ligand-benchmark/blob/a0b67b93c4f12f63a8bb417a1f6e4df4dc10c6c5/plbenchmark/utils.py#L85C1-L85C87